bears/Constants.py

```diff
@@ -4,3 +4,36 @@
 
 # Path to the bears directory
 bears_root = os.path.dirname(__file__)
+
+# Constants to be used for the uploadTool to improve automation
+
+classifiers = """[
+              'Development Status :: 4 - Beta',
+
+              'Environment :: Console',
+              'Environment :: MacOS X',
+              'Environment :: Win32 (MS Windows)',
+              'Environment :: X11 Applications :: Gnome',
+
+              'Intended Audience :: Science/Research',
+              'Intended Audience :: Developers',
+
+              'Programming Language :: Python :: Implementation :: CPython',
+              'Programming Language :: Python :: 3.3',
+              'Programming Language :: Python :: 3.4',
+              'Programming Language :: Python :: 3.5',
+              'Programming Language :: Python :: 3 :: Only',
+
+              'Topic :: Scientific/Engineering :: Information Analysis',
+              'Topic :: Software Development :: Quality Assurance',
+              'Topic :: Text Processing :: Linguistic']"""
+imports = """
+import locale
+
+import setuptools.command.build_py
+from setuptools import find_packages, setup"""
```

Comment:
please speak with @mr-karan he does some templating as well and we need to make sure everything is less diverse